packages/spacecat-shared-data-access/src/index.d.ts

```diff
@@ -722,6 +773,12 @@ export interface DataAccess {
       jobId: string,
       status: string,
     ) => Promise<ImportUrl[]>;
+  getApiKeyById: (
+      id: string,
```

Comment:
Will we be querying by id, or by the hashed key value computed from the `x-api-key` header?